tests/test_plotfileparticledata.py

```diff
@@ -0,0 +1,50 @@
+from dataclasses import dataclass
+
+import amrex.space3d as amr
+
+if not amr.initialized():
+    amr.initialize([])
```

Comment:
The `tests/conftest.py` file does that.

Reply:
Remove the lines to get rid of the `already initialized` error:
```suggestion
```

Reply:
Oh! Pytest's verbose mode was being highly misleading: it must have crashed just after printing `tests/test_aos.py` which made it look like the error was coming from there, when it was coming from somewhere else.

So why does this cause an error? If `amr.initialize` is being guarded. I like this approach because then a relatively inexperienced user can run the test without needing to know what pytest does behind the scenes.

I.e. I would prefer to have similar behaviour when it comes to initializing as `mpi4py`

Reply:
Ah! So the problem was that all tests are imported as modules. So my test was initializing AMReX before pytest was. 

* I added guard against this here: https://github.com/AMReX-Codes/pyamrex/pull/342/commits/fa20a6fb89038fe1131a6f0a3c60773648cf1c01

* And I am guarding against initializing pyamrex before the fixture does here: https://github.com/AMReX-Codes/pyamrex/pull/342/commits/04c01427713f7f8b4e5c1b891ad9749c7886bb59

This lets a user build off the test as an example without needing to know that pytest autmatically initialized amrex.

What do you think @ax3l ?

Reply:
Thanks! Yes, when I learned pytest the other year I was also having the same urge initially. Nonetheless by now, since this adds more overhead for tests to maintain and they can literally be called with the documented one-liner like an individual script + the unification of options we want for all tests that we put in `conftest.py` I would keep it simple, pytest only.